src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/Interop/DWrite/IDWriteFactory.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace MS.Internal.Interop.DWrite
+{
+    internal unsafe struct IDWriteFactory : IUnknown
+    {
+        public void** lpVtbl;
+
+        public int QueryInterface(Guid* riid, void** ppvObject)
+        {
+            return ((delegate* unmanaged[Stdcall]<IDWriteFactory*, Guid*, void**, int>)(lpVtbl[0]))((IDWriteFactory*)Unsafe.AsPointer(ref this), riid, ppvObject);
```

Comment:
`[Stdcall]` is default call convention, and no need to specify it here.

No need to use explicit types like `IDWriteFactory*` where you can just have `IntPtr` or `void*`

Reply:
Fixed in 557eea0394c625106e6d7f87fb35426987b47277. I tweaked ClangSharp to not add Stdcall.